Samples~/AvatarCreatorSamples/AvatarCreatorElements/Scripts/SimpleAvatarCreator.cs

```diff
@@ -220,5 +244,19 @@ private void SetupAvatar()
                 { BodyType = bodyType, OutfitGender = gender }, animator);
             animator.runtimeAnimatorController = animationController;
         }
+
+        public async void OnAvatarDeletion(string avatarId)
```

Comment:
We used OnAvatarCreated, OnAvatarSelected so lets keep the naming consistent and use OnAvatarDeleted 
```suggestion
        public async void OnAvatarDeleted(string avatarId)
```